src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -73,7 +73,6 @@ export function createEuiListItem({
       }
 
       if (
-        !link.externalLink && // ignore external links
```

Comment:
Is this change by intention? Seems the external link will pass here.

Reply:
Yes. `link.externalLink` logic was previously added to enable workspace switch in left menu, which is no longer needed.